packages/manager/src/features/Linodes/LinodeCreate/AdditionalOptions/Alerts/Alerts.tsx

```diff
@@ -28,6 +29,19 @@ export const Alerts = () => {
     field.onChange(updatedAlerts);
   };
 
+  const subHeading = isAclpAlertsPreferenceBeta ? (
+    <>
+      Receive notifications through System Alerts when metric thresholds are
+      exceeded. After you've created your Linode, you can create and manage
+      associated alerts on the <strong>centralized Alerts</strong> page.{' '}
+      <Link to="https://techdocs.akamai.com/cloud-computing/docs/configure-email-alerts-for-resource-usage-on-compute-instances">
+        Learn more.
+      </Link>
```

Comment:
```suggestion
      <Link to="https://techdocs.akamai.com/cloud-computing/docs/configure-email-alerts-for-resource-usage-on-compute-instances">
        Learn more
      </Link>.
```

Let's keep the period outside of the `Link`